### PR TITLE
[refactor]&[fix] `wordpress-{openshift-,}namespace` role

### PR DESCRIPTION
**Pull request by domq**
- Rename `wordpress-openshift-namespace` role to `wordpress-namespace`
- Move “namespacey” tasks from the `rke2-cluster` role into the new role
- Conversely, spin out a new `openshift-reverse-proxy` role for the “prod-only,” temporary stuff
- Completely rewrite secret management, so that `eyaml_keys` is inside the secret vars files
- Make it all almost¹ work for `-t wp`, which serves as the validation for the refactor operation
- Implement multi-Kubernetes login as-code, as a prerequisite for the previous step

¹ Barring unrelated issues such as [WPN-218](https://go.epfl.ch/WPN-218)

[WPN-218]: https://erpdev.atlassian.net/browse/WPN-218?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by domq** [comment]
💡 This pull request introduces new dependencies (see changes to `ansible/requirements.txt`); in order to test it, you should invalidate your suitcase.

File: ansible/inventory/test/inventory-test.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 all:
   vars:
     ansible_user: root
-    wpn_namespace: wordpress-test
+    rke2_cluster_name: wordpress-test
```
Comment (obieler) [inline ansible/inventory/test/inventory-test.yml:5]: Please put this in another pull request.

Comment (domq) [reply ansible/inventory/test/inventory-test.yml]: Rebased this one out, thanks!

**Review by ponsfrilus — APPROVED** [review]
:+1:

**Review by obieler — APPROVED** [review]

**Review by ponsfrilus — APPROVED** [review]

File: ansible/roles/wordpress-namespace/tasks/menu-api.yml [hunk 2]
```diff
@@ -3,11 +3,11 @@
 
 - name: "Testing ImageStreams and their build information for Menu API"
   when: >-
-    openshift_namespace == "wwp-test"
+    inventory_namespace == "wwp-test"
```
Comment (rosamaggi) [inline ansible/roles/wordpress-namespace/tasks/menu-api.yml:6]: The namespaces `wwp` nd `wwp-test  dont exist anymore so this code should be reviewed.

File: ansible/vars/secrets-wwp-infra.yml [hunk 3]
```diff
@@ -26,11 +26,3 @@ s3_backup:
       secret: ENC[PKCS7,MIIBmQYJKoZIhvcNAQcDoIIBijCCAYYCAQAxggEhMIIBHQIBADAFMAACAQEwDQYJKoZIhvcNAQEBBQAEggEAReSVtx4lzjHPiUmTKgD6zEDT/SrT/vzuo1A7WFvGtKbjnqZ1p7quEj30RCMfcneDB1JJ3xL7wI46L2yXSbqcYoaJYbp4eZov7FCyPImrsyGsPbNOZdGFjUhSBOCJ/3YqKXlXHauk0e/p9W/MrwqpDoVrkDcf7cu/UpXUF02QawowjGNEqXsrA30tbJ2pn1hjwMJvrG9WiFsLr3TAc/f5t9uo3TWuBJd4A7PLlOcQldu/JhAzt1CG/JjhpRlkSZcWtC9ylEkzG8OqyzrFRidI367md5Qh0sRBFpZ1eRkCdC2UppgbI9OgKt/TgHPL37UhkevFElJdqu4kK46QovXISjBcBgkqhkiG9w0BBwEwHQYJYIZIAWUDBAEqBBAC2HuBqQPn61NSZyWQ+7hdgDCxizQpZJwqOjT5wfKNN+ofwVAPwiQlRptKbguWyntLgywMOtot27BsyxIaepA2gjE=]
```
[the remark this answers is not in the document]
Comment (domq) [reply ansible/vars/secrets-wwp-infra.yml]: Right now they are dead code indeed; my thought was to put them back in use once we decide how to do backups and restore in the new architecture. I just pushed a commit that deletes them (we can always retrieve them from `master` if needed).

File: ansible/vars/secrets-wwp-test.yml [hunk 4]
```diff
@@ -39,23 +39,3 @@ s3_backup_prod:
       secret: ENC[PKCS7,MIIBmQYJKoZIhvcNAQcDoIIBijCCAYYCAQAxggEhMIIBHQIBADAFMAACAQEwDQYJKoZIhvcNAQEBBQAEggEAXjRoEGQyHfPrzcwCIEt6mmnXzZhClkAMcCJK5jv1z+K1+NHSfKfGF2h5qCSfaoZuT+rSFxVb4Vx1D5AR4WBrGIzcCN0hwTR5egKae9BmhwEoQLxWMNoYIcsfdoaPwK3W9vKKfC0tnkjCfq3Z21edQTIYJLKl5b+ZW0XGZXwuzAez+gqQ9mVL3V+zXFeH5EtLnnL2YNBdmvZhxShl5Clv68P3Fu8NLMypuiglzC7qwDoAVzRhwOWtjCEqRMGvnBZFRdqDHvC6gzOEjnvJR9THTvZl96L53r9a3vVLSUeZg8pU/RwzLYT6KdPbvEWQfiazEfjUN9vQUK0xAojwnPEgITBcBgkqhkiG9w0BBwEwHQYJYIZIAWUDBAEqBBBAmIpFv6fJnoladcTxPJV2gDCZQxgVypnBv7j3Ce6c8JlhTxZVWkBQQwyRVLWO5cU5Gl5L5BBnlGDGexlybxUJo60=]
```
[the remark this answers is not in the document]
Comment (domq) [reply ansible/vars/secrets-wwp-test.yml]: Done.
